server/pulp/server/db/migrations/0024_distributor_last_updated.py

```diff
@@ -15,4 +16,9 @@ def migrate(*args, **kwargs):
             distributor[key] = distributor['last_publish']
```

Comment:
This first `continue` will prevent you to check the `key2`, you will just move on to the next distributor.
My suggestion is not to use `continue` at all in this case. How about this:
```
if 'last_publish' in distributor:
    distributor[key] = distributor['last_publish']
elif key not in distributor:
    distributor[key] = dateutils.now_utc_datetime_with_tzinfo()
```
Does it make sense or maybe I am missing something?